include/mender-os.h

```diff
@@ -1,5 +1,5 @@
 /**
- * @file      mender-scheduler.h
+ * @file      mender-os.h
  * @brief     Mender scheduler interface
```

Comment:
Should it be?
```suggestion
 * @brief     Mender OS interface
```
If so, also change in platform specific header files

Reply:
Another good catch by Larsito! :beers:

Reply:
:beers: